### PR TITLE
setting lgtm_acts_as_approve: false for all our repos

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

## This introduces a breaking change

- [x] Yes
- [ ] No

@HumairAK @tumido @harshad16 

/sig devops

### CODE REVIEW

File: prow/overlays/cnv-prod/plugins.yaml [hunk 1]
```diff
@@ -87,6 +87,7 @@ approve:
       - thoth-station
     require_self_approval: true
     ignore_review_state: false
+    lgtm_acts_as_approve: false
```
Comment (tumido) [inline prow/overlays/cnv-prod/plugins.yaml:90]: @HumairAK @anishasthana @4n4nd we want this for operate-first as well! :smile:

Comment (goern) [reply prow/overlays/cnv-prod/plugins.yaml:90]: you got it, if we merge it ;)

**Review by tumido — APPROVED** [review]

**Review by 4n4nd — APPROVED** [review]

**Pull request merged**